website/static/js/pages/shareEmbedConfig.js

```diff
@@ -0,0 +1,7 @@
+var pym = require('pym.js');
```

Comment:
Yep, but prefer `npm` to `bower`. We are trying to reduce/remove bower packages (rationale: https://webpack.github.io/docs/usage-with-bower.html#prefer-modules-from-npm-over-bower)